<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgKeyPaste</class>
 <widget class="QDialog" name="DlgKeyPaste">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Special paste to a range of keys</string>
  </property>
  <widget class="QSpinBox" name="spinBox1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PASTE from - to KEY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtOk">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>PASTE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtCn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SET ALL</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_C1_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Color Text</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_C2_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Color Fill</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_C3_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Color Outline</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>having the same</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_DS_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Design</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_FS_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Font/Align/Icon</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>181</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>spinBox1</zorder>
  <zorder>spinBox2</zorder>
  <zorder>label</zorder>
  <zorder>BtOk</zorder>
  <zorder>BtCn</zorder>
  <zorder>checkBox</zorder>
  <zorder>checkBox_C1_2</zorder>
  <zorder>checkBox_C2_2</zorder>
  <zorder>checkBox_C3_2</zorder>
  <zorder>label_2</zorder>
  <zorder>checkBox_DS_2</zorder>
  <zorder>checkBox_FS_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
